package.json

```diff
@@ -3,11 +3,8 @@
   "version": "1.0.0",
   "description": "Loads environment variables from .env file",
   "main": "lib/main.js",
-  "directories": {
-    "test": "test"
-  },
   "scripts": {
-    "test": "gulp ci"
+    "test": "lab test/* --coverage && standard"
```

Comment:
if we're using lab for its built-in coverage why not also use its built in linter?